### PR TITLE
Access modifiers

### PR DESCRIPTION
**Pull request by VargBurz**
This PR updates access modifiers in the `Pod`.

### Changes:
   - rename `_series`, `_options`, `_d3` to `series`, `options`, `d3`
   - add `protected` and `readonly` modifiers to `series`, `options`, `d3`, `el` fields  
   - add `_` symbol in the `private` `clipPathUID` field name
   
### Note:
   - use `_d3` arg in `Pod` constructor  to avoid an error: 'd3' is referenced directly or indirectly in its own type annotation

### CODE REVIEW

**Review by rozetko — APPROVED** [review]

File: dist/index.d.ts [hunk 1]
```diff
@@ -4,9 +4,9 @@ import { Margin, TimeSerie, Options, TickOrientation, TimeFormat, ZoomOrientatio
 import { palette } from './colors';
 import * as d3 from 'd3';
 declare abstract class ChartwerkPod<T extends TimeSerie, O extends Options> {
-    protected _d3: typeof d3;
-    protected _series: T[];
-    protected readonly _options: O;
+    private readonly _d3;
```
Comment (jonyrock) [inline dist/index.d.ts:7]: it should be protected.
if you make it readonly, then you can kind of guaranty that it's not going to use badly (you can enforce "readonlines" by typescipt, but let's not think about that for now)

File: src/index.ts [hunk 2]
```diff
@@ -122,6 +127,22 @@ abstract class ChartwerkPod<T extends TimeSerie, O extends Options> {
...
+
+  protected get options(): O {
+    return this._options;
+  }
+
+  protected get d3(): typeof d3 {
+    return this._d3;
+  }
+
+  protected get el(): HTMLElement {
```
Comment (jonyrock) [inline src/index.ts:142]: there is no point to make protected getters if you have protected readonly feilds

**Review by jonyrock — CHANGES_REQUESTED** [review]

File: src/index.ts [hunk 3]
```diff
@@ -76,13 +76,14 @@ abstract class ChartwerkPod<T extends TimeSerie, O extends Options> {
...
-  private readonly _options: O;
+  protected readonly options: O;
+  protected readonly d3: typeof d3;
 
   constructor(
     // maybe it's not the best idea
-    private readonly _d3: typeof d3,
-    private readonly _el: HTMLElement,
-    private readonly _series: T[] = [],
+    _d3: typeof d3,
```
Comment (jonyrock) [inline src/index.ts:84]: why underscore here?

Comment (VargBurz) [reply src/index.ts:84]: Use underscore here to avoid an error: 'd3' is referenced directly or indirectly in its own type annotation.
Added a Note for it

**Review by jonyrock — APPROVED** [review]

**Pull request merged**